iOS/Schema/Sources/Action/Types/Navigate/Navigate.swift

```diff
@@ -14,24 +14,37 @@
  * limitations under the License.
  */
 
+/// Handles screens navigations actions of the application.
 public enum Navigate: RawAction {
-    
+    /// Opens up an available browser on the device and navigates to a passed URL as String.
     case openExternalURL(String)
+    /// Opens a screen that is defined all locally in your app (does not depend on Beagle) which will be retrieved using `DeeplinkScreenManager`.
```

Comment:
I found this sentence a little confusing when reading, maybe a better way would be something like: 
```suggestion
  /// Opens a screen that is defined completely local in your app (does not depend on Beagle) which will be retrieved using `DeeplinkScreenManager`.
```